test/integration/commands/pluck_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Integration.Commands.PluckTest do
+  use Cog.AdapterCase, adapter: "test"
+
+  setup do
+    user = user("lucky", first_name: "Shamaus", last_name: "McLucky")
+    |> with_chat_handle_for("test")
+
+    {:ok, %{user: user}}
+  end
+
+  test "with no field option", %{user: user} do
```

Comment:
I agree @christophermaier. Decoupling the testing from the templates is a much better route. I only mention this here because I had such a hard time separating templating from command output when I was writing the alias and which command tests.